redi-connect-front/src/components/organisms/Checklist.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { Section, Container, Element, Columns } from 'react-bulma-components'
+import Heading from '../atoms/Heading'
+import { ReactComponent as Check } from '../../assets/images/checkmark.svg'
```

Comment:
Please do the checkmark without an SVG. Check out the implementation for the checkbox.

Reply:
Also, if you need a variable called `check`, please make it more explicit

Reply:
done, image deleted, all css now